lib/iris/tests/runner/_runner.py

```diff
@@ -127,7 +127,9 @@ def finalize_options(self):
         if self.stop:
             print('Stopping tests after the first error or failure')
         if self.num_processors is None:
-            self.num_processors = multiprocessing.cpu_count() - 1
+            # Choose a magic number that works reasonably well for the default
+            # number of processes.
+            self.num_processors = (multiprocessing.cpu_count() + 1) // 4 + 1
```

Comment:
@pelson What's the logic behind this lovely piece of maths?

Reply:
It's a magic number... there is no solid/defensible logic other than:

* It should be at least one
* It shouldn't fill all CPUs if there are lots of them

:smile: